### DISCUSSION TITLE
Error while trying to install packages

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2306] [Q&A] [answered]
GM GM
***
![image](https://github.com/user-attachments/assets/4154c130-91c4-4e2b-9bf2-58a31d8cb986)
***
What should i do ?

**Comment by EngrPips** [comment, marked as answer]
Hello @MostPow3rful, Delete everything inside of your `lib` folder then rerun `make install`

**Reply by MostPow3rful** [reply]
Oh. Thanks <3

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it
